app/__init__.py

```diff
@@ -22,12 +19,10 @@ def create_app(config_name):
     application.wsgi_app = ProxyFix(application.wsgi_app)
     application.config.from_object(config[config_name])
 
-    for name in config_attrs(config[config_name]):
-        if name in os.environ:
-            application.config[name] = convert_to_boolean(os.environ[name])
```

Comment:
To immortalise the conversation we just had in real life. The config logic is one of the things we should move into the utils package.